From 26acd6780f952534a1557c5c403072ad82537e65 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Fri, 23 Oct 2020 05:59:51 -0700
Subject: [PATCH 1724/1851] dt: bindings: media: xilinx: sdirxss : Add optional
 PICXO gpio handle

commit b1387820d8f1bb4fa167566efc42d7d1092e35ec from
https://github.com/Xilinx/linux-xlnx.git

Add an optional gpio reset handle for PICXO module. This is active high.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/media/xilinx/xlnx,sdirxss.txt          | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt
index 169338ed086d..0a93f29fdbe0 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt
@@ -39,6 +39,8 @@ Optional properties:
 
 - reset_gt-gpios: contains GPIO reset phandle for FMC init done pin in GT.
   This pin is active low.
+- picxo_reset-gpios: contains GPIO reset phandle for PICXO done pin for PICXO module.
+  This pin is active high.
 - xlnx,bpp: This denotes the bit depth as 10 or 12 based on IP configuration.
   The default value is 10 for backward compatibility.
 
@@ -55,6 +57,7 @@ Example:
 			clocks = <&clk_1>, <&si570_1>, <&clk_2>;
 			clock-names = "s_axi_aclk", "sdi_rx_clk", "video_out_clk";
 			reset_gt-gpios = <&axi_gpio_0 0 0 GPIO_ACTIVE_LOW>;
+			picxo_reset-gpios = <&axi_gpio_0 2 0 GPIO_ACTIVE_HIGH>;
 
 			ports {
 				#address-cells = <1>;
-- 
2.31.1

